training-react/finger-exercises/ecmascript/01 - spread/index.js

```diff
@@ -0,0 +1,37 @@
+import { isArray } from './utils';
+
+export function min(param, ...arg) {
+  let val;
+  if (param !== undefined && param !== null) {
```

Comment:
You can replace this condition for `if (param)` see this [post](https://www.sitepoint.com/javascript-truthy-falsy/)